apps/angular/src/app/features/anime/anime-table/anime-table.component.ts

```diff
@@ -0,0 +1,251 @@
+import { ChangeDetectionStrategy, Component, OnDestroy, OnInit } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { PageEvent } from '@angular/material/paginator';
+import { MatSelectChange } from '@angular/material/select';
+import { Sort } from '@angular/material/sort';
+import { ActivatedRoute, Router } from '@angular/router';
+import { TypeDto } from '@js-camp/core/dtos';
+import { Anime, Pagination, SortValue } from '@js-camp/core/models';
+import { BehaviorSubject, combineLatestWith, debounceTime, distinctUntilChanged, ignoreElements, map, merge, Observable, shareReplay, skip, startWith, Subject, switchMap, takeUntil, tap } from 'rxjs';
+
+import { DEFAULT_ACTIVE_PAGE, DEFAULT_SEARCH, FILTER_TYPE_OPTIONS, ORDERING_OPTIONS, OrderOption, SORT_OPTIONS } from '../../../../constants';
+import { AnimeService, QueryUrl, SettingOfAnimeList } from '../../../../core/services';
+
+/** Anime table list. */
+@Component({
+  selector: 'camp-anime-table',
+  templateUrl: './anime-table.component.html',
+  styleUrls: ['./anime-table.component.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+
+})
+export class AnimeTableComponent implements OnInit, OnDestroy {
+
+  /** Column of table. */
+  public displayedColumns: string[] = ['image', 'titleEnglish', 'titleJapan', 'airedStartDate', 'type', 'status'];
```

Comment:
```suggestion
  public displayedColumns: readonly string[] = ['image', 'titleEnglish', 'titleJapan', 'airedStartDate', 'type', 'status'];
```

Reply:
Or you can use safer construction:
```suggestion
  public displayedColumns = ['image', 'titleEnglish', 'titleJapan', 'airedStartDate', 'type', 'status'] as const;
```